integrations-and-supported-tools/great_expectations/notebooks/Neptune_Great_Expectations.ipynb

```diff
@@ -297,7 +297,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The above code cell logs the GX Expectations suite name to the context configuration to the `gx/expectations/expectations_suite_name` field of the run, and creates a numbered folder for each expectation in the `gx/expectations` namespace."
+    "The above code cell does two things:
+    * logs the GX Expectations suite name to the context configuration to the `gx/expectations/expectations_suite_name` field of the run,
+    * creates a numbered folder for each expectation in the `gx/expectations` namespace."
```

Comment:
It can be treacherous to edit `.ipynb` source directly :) Plain multi-line Markdown doesn't work.

![image](https://github.com/neptune-ai/examples/assets/7567953/106defa2-6be8-46eb-ae7d-36b6c62261e2)

Siddhant already fixed it, but you can try to use JSON syntax with `\n` and the like, or check out the notebook and apply your edits locally 👌🏻

Reply:
My bad, sorry!